code/_globalvars/bitfields.dm

```diff
@@ -181,6 +181,22 @@ DEFINE_BITFIELD(item_flags, list(
 	"NO_PIXEL_RANDOM_DROP" = NO_PIXEL_RANDOM_DROP,
 ))
 
+DEFINE_BITFIELD(flags_inv, list(
+	"HIDEGLOVES" = HIDEGLOVES,
+	"HIDESUITSTORAGE" = HIDESUITSTORAGE,
+	"HIDEJUMPSUIT" = HIDEJUMPSUIT,
+	"HIDESHOES" = HIDESHOES,
+	"HIDEMASK" = HIDEMASK,
+	"HIDEEARS" = HIDEEARS,
+	"HIDEEYES" = HIDEEYES,
+	"HIDEFACE" = HIDEFACE,
+	"HIDEHAIR" = HIDEHAIR,
+	"HIDEFACIALHAIR" = HIDEFACIALHAIR,
+	"HIDENECK" = HIDENECK,
+	"HIDEHEADGEAR" = HIDEHEADGEAR,
+	"HIDESNOUT" = HIDESNOUT,
```

Comment:
if you could abc-sort these please, i did this when i first made this file but not many people kept it up

Reply:
theres a vsc extension to bind this to f9 that i realize most people dont have and i should probably advertise